dg/media/app_dashboards/js/loop_dashboard.js

```diff
@@ -597,7 +664,12 @@ function set_filterlistener() {
             $('#aggregator_payment_details').hide();
             $('#payments_to_date').prop('disabled', false);
             var from_date = new Date(new Date(start_date));
-            $('#payments_to_date').val(from_date.getFullYear() + "-" + (from_date.getMonth() + 1) + "-" + (from_date.getDate() + 15));
+            var daysInMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+            if (from_date.getDate() >= 16 && daysInMonth[from_date.getMonth()] != (from_date.getDate() + 15)) {
```

Comment:
isn't there any ready made function or library available that we can use instead of handling date thing at our end ?